okhttp/src/jvmMain/kotlin/okhttp3/internal/connection/RoutePlanner.kt

```diff
@@ -53,11 +53,11 @@ interface RoutePlanner {
 
   fun trackFailure(e: IOException)
 
-  /**
-   * Returns true if the current route has a failure that retrying could fix, and that there's
-   * a route to retry on.
-   */
-  fun retryAfterFailure(): Boolean
+  /** Returns true if the current route has a failure that retrying could fix. */
```

Comment:
hasRetryableFailure based on the comment?

Reply:
Fixed:
```
  /** Returns true if this planner has received any failures. */
```